packages/venia-concept/src/util/makeMediaUrl.js

```diff
@@ -0,0 +1,37 @@
+import memoize from 'lodash/memoize';
```

Comment:
We pull in [lodash.debounce](https://github.com/magento-research/pwa-studio/blob/release/2.0/package.json#L69) as its own dependency, should we do that for `memoize` as well?